docs/spec/spec.md

```diff
@@ -845,7 +845,7 @@ grpc:ClientBasicAuthHandler handler = new (config);
 map<string|string[]>|grpc:ClientAuthError result = handler.enrich(requestHeaders);
 ```
 
-#### 5.1.2.6 Client - Bearer Token Auth
+#### 5.1.2.6 Client - bearer token auth
```

Comment:
```suggestion
#### 5.1.2.6 Client - bearer token auth
```
Check the industrial terms.

Reply:
https://datatracker.ietf.org/doc/html/rfc6750